test/flake.nix

```diff
@@ -23,26 +23,23 @@
         inputs.check-flake.flakeModule
       ];
       perSystem = { self', pkgs, ... }: {
+        haskellFlakeProjectModules.default = {
```

Comment:
@roberth I must say I'm not happy with this flake-parts module now providing *two* (not one) top-level options -- `haskellProjects` and, now, `haskellFlakeProjectModules`.

A single top-level would make sense. Like:

```nix
{ 
  haskellFlake = {
    projectModules = { .. };
    projects.default = { .. };
  };
}
```

It also makes the option name obvious enough to indicate that it comes from `inputs.haskell-flake` rather than any other inputs (the same cannot be strictly said for `haskellProjects` naming). A similar naming is adopted by other modules like https://github.com/Platonic-Systems/mission-control

wdyt?

Reply:
OTOH, the current way of naming it `outputs.haskellFlakeProjectModules.${system}.default` appears to follow the unspoken convention of using flat attributes à la `nixosModules`, `darwinModules`, etc.

Reply:
I think we need options in two separate locations anyway. See previous comment https://github.com/srid/haskell-flake/pull/79/files#r1103655549

> flat attributes à la `nixosModules`,

Flat attributes is probably the way to go. I've made an exception for `flakeModule`, which I'm not eager to remove until the `imports` UX is improved. Not sure exactly what can be achieved in that area yet. My previous attempt found too much resistance, so I'll have to find something creative... Anyway, until then, I'd say flat is good.